extensions/typescript-language-features/package.nls.json

```diff
@@ -73,9 +73,10 @@
 	"typescript.suggestionActions.enabled": "Enable/disable suggestion diagnostics for TypeScript files in the editor.",
 	"typescript.preferences.quoteStyle": "Preferred quote style to use for quick fixes: `single` quotes, `double` quotes, or `auto` infer quote type from existing imports.",
 	"typescript.preferences.importModuleSpecifier": "Preferred path style for auto imports.",
-	"typescript.preferences.importModuleSpecifier.auto": "Automatically select import path style. Prefers using a relative import if `baseUrl` is configured and the relative path has fewer segments than the non-relative import.",
-	"typescript.preferences.importModuleSpecifier.relative": "Relative to the file location.",
-	"typescript.preferences.importModuleSpecifier.nonRelative": "Based on the `baseUrl` configured in your `jsconfig.json` / `tsconfig.json`.",
+	"typescript.preferences.importModuleSpecifier.shortest": "Prefers using a non-relative import only if one is available that has fewer path segments than a relative import.",
+	"typescript.preferences.importModuleSpecifier.relative": "Prefers using a relative path to the imported file location.",
+	"typescript.preferences.importModuleSpecifier.nonRelative": "Prefers using a non-relative import based on the `baseUrl` or `paths` configured in your `jsconfig.json` / `tsconfig.json`.",
+	"typescript.preferences.importModuleSpecifier.projectRelative": "Prefers using a non-relative import only if the relative import path would leave the package or project directory.",
```

Comment:
If `projectRelative` is a new setting, could you also note which TS version is required to use it?